ios/RNDateTimePickerManager.m

```diff
@@ -48,7 +48,9 @@ - (UIView *)view
   } else {
     UIColor* defaultColor;
     if (@available(iOS 13.0, *)) {
+#ifdef __IPHONE_13_0
```

Comment:
did you test this with xcode 10? Shouldn't the `ifdef` wrap the entire if? like here https://stackoverflow.com/a/58083853/2070942

Reply:
No I'm with Xcode 11, but I think you are right, I'll download Xcode 10 and test this, thanks

Reply:
I don't think you need to go through that hassle, I was just asking. Frankly, I don't think we're required to support xcode 10, it's already 6 months old. I think if someone needs that support, they should contribute it, we should not be spending hours by downloading xcode 10...

Reply:
I agree. So we need to update the readme requiring use of Xcode 11, what do you think?

Reply:
yes, that sounds good. :)

Reply:
Hello, @vonovak , I tested on XCode 10 and worked fine.